_rules/video-description-track-f196ce.md

```diff
@@ -97,8 +97,8 @@ A video element that is not visible on the page.
 
 ```html
 <video controls style="display: none;">
-	<source src="../test-assets/rabbit-video/video.mp4" type="video/mp4" />
-	<source src="../test-assets/rabbit-video/video.webm" type="video/webm" />
+	<source src="../test-assets/rabbit-video/video.mp4" type="video/mp4">
+	<source src="../test-assets/rabbit-video/video.webm" type="video/webm">
 	<track
```

Comment:
`<track>` is also a void element, so it shouldn't have the end slash, similar to `<source>`